.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
-dist/
\ No newline at end of file
+dist/
+config/
```

Comment:
This will ignore the config folder itself. Ignore specific secret files only. Also, create a `filename.sample.js` which has same keys but dummy values and this file should not be ignored.